code/index.html

```diff
@@ -1,22 +1,163 @@
+        <article>
+          <h2>
+            Ideas to plan for a great trip
+          </h2>
+          <p>
+            Planning a trip overseas, whether long or short, can seem overwhelming. But it doesn’t have to be! In this
+            article, we’ll show you how to take the stress out of planning a trip, by laying out a simple step-by-step
+            guide that’ll ensure you don’t forget anything. </p>
+          <button class="featured-article-button">Read More</button>
+        </article>
+      </div>
+      <div class="flex-container-2">
+        <img src="img/rolands-varsbergs-miKmVyq3qhE-unsplash.jpg" alt="article-1">
+      </div>
     </section>
 
-    <section class="">
-      <!-- The other news  -->
+    <div class="popular-destinations-header">
+      <h2>Popular Destinations</h2>
+    </div>
+
+    <!-- Responsive flexbox with:
+      - 4 columns on laptop
+      - 3 on tablet
+      - 1 on mobile -->
+    <section class="popular-destinations">
+      <div class="flex-container-4">
+        <article class="news-article">
+          <img src="img/greece-ryan-spencer-XGKaRnWjv1c-unsplash.jpg" alt="article-1">
+          <div class="widget">
+            <span>Sunny</span>
+            <span>+35°</span>
+          </div>
+          <div class="news-article-child black">
+            <p class="tag">| SUMMER |</p>
+            <h3>Greece</h3>
+          </div>
+        </article>
+      </div>
+      <div class="flex-container-4">
+        <article class="news-article">
+          <img src="img/sweden-linus-mimietz-QYI__0vehUs-unsplash.jpg" alt="article-1">
+          <div class="widget">
+            <span> <i class="fa-solid fa-sun-bright"></i></span>
+            <span>Rainy</span>
+            <span>-2°</span>
+          </div>
+          <div class="news-article-child">
+            <p class="tag">| WINTER |</p>
+            <h3>Sweden</h3>
+          </div>
+        </article>
+      </div>
+      <div class="flex-container-4">
+        <article class="news-article">
+          <img src="img/iceland-jonny-auh-z99iWocuDt0-unsplash.jpg" alt="article-1">
+          <div class="widget">
+            <span>Snowy</span>
+            <span>-10°</span>
+          </div>
+          <div class="news-article-child">
+            <p class="tag">| WINTER |</p>
+            <h3>Iceland</h3>
+          </div>
+        </article>
+      </div>
+      <div class="flex-container-4">
+        <article class="news-article">
+          <img src="img/france2-cyril-mazarin-WSvth_lwCi0-unsplash (1).jpg" alt="article-1">
+          <div class="widget">
+            <span>Cloudy</span>
+            <span>+15°</span>
+          </div>
+          <div class="news-article-child">
+            <p class="tag">| SPRING |</p>
+            <h3>France</h3>
+          </div>
+        </article>
+      </div>
     </section>
-  </body>
-</html>
+
+    <!-- Footer section with responsive flexbox -->
+    <div class="footer">
+      <div class="footer-left">
```

Comment:
Maybe confusing class names if you start with mobile first and the three `div`s are placed in a column?

Reply:
Just saw those Ida!! Thanks, your comment makes sense